src/packerlicious/builder.py

```diff
@@ -800,6 +800,54 @@ class HyperV(PackerBuilder):
         'temp_path': (str, False),
     }
 
+class HyperVvmcx(PackerBuilder):
+    """
+    Hyper-V Builder (from a vmcx)
+    https://www.packer.io/docs/builders/hyperv-vmcx.html
+    """
+    resource_type = "hyperv-vmcx"
+
+    props = {
+        'clone_from_vmxc_path': (str, True),
+        'clone_from_vm_name': (str, True),
```

Comment:
`clone_from_vmxc_path` and `clone_from_vm_name` should both be required = `False`